safe/impact_functions/earthquake/itb_earthquake_fatality_model/impact_function.py

```diff
@@ -115,7 +115,7 @@ def __init__(self):
                 1: 0, 2: 0, 3: 0, 4: 0, 5: 0, 6: 1.0,
                 7: 1.0, 8: 1.0, 9: 1.0, 10: 1.0
             }),
-            ('mmi_range', range(2, 10)),
+            ('mmi_range', range(4, 11)), # from MMI 4 to 10
```

Comment:
If the information is required, I have no objection. I can change it accordingly then.
Another thing is that I wonder whether exposed number can be pulled within +/-0.25 MMI which is more consistent with other records such as EXPO-CAT. For instance 4.75-5.25 is rounded to 5.0.